alts/src/main/java/io/grpc/alts/AltsContext.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.alts;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.grpc.alts.internal.AltsInternalContext;
+import io.grpc.alts.internal.SecurityLevel;
+
+/** {@code AltsContext} contains security-related information on the ALTS channel. */
+public class AltsContext {
+
+  private final AltsInternalContext wrapped;
+
+  AltsContext(AltsInternalContext wrapped) {
+    this.wrapped = wrapped;
+  }
+
+  @VisibleForTesting
```

Comment:
Is this for testing this class or for letting other classes testing themselves?

Reply:
This should make classes that use the AltsContext testable. However, without parameters this isn't very useful. Therefore I've changed the method to specify the service accounts (security level is fixed anyway).

Reply:
While security level is fixed today, you can't actually test any code that looks at it if it is hard-coded during testing. It seems like maybe we should just suck it up and make a builder fascade as well, that delegates to HandshakerResult's builder. @jiangtaoli2016, thoughts?

I'm actually fairly okay with this as it stands, if we do want to leave it as-is. We tend to limit VisibleForTesting usage to instances where the project's own unit tests need access. For cases like this, we'd just call this "a public API". Let's remove the VisibleForTesting annotation here? I honestly don't see much harm someone could do, even using it in production.

Reply:
I am fine with it, since AltsContext is just a wrapper of AltsInternalContext. This fixed security level is only exposed for testing.

Reply:
I've removed the VisibleForTesting annotation.